### DISCUSSION TITLE
Deep wounds behavior related to non-physical critical strike

### DISCUSSION THREAD

**Discussion by LiaohuChromie** [chromiecraft/chromiecraft#4213] [Suggestions]
Hello ! 

Recently it was re-discovered on classic retail that deep wounds ( https://www.wowhead.com/wotlk/spell=12867/deep-wounds) was able to proc from ANY critical strike, including non-physical-melee ones. (Such as using fiery weapon enchant, fiery gauntlet, blazefury medaillon, stormchops etc, you get the idea). 

Even though it was hotfixed by blizzard, they made a blue post stating it was NOT a bug, but a change and it was like that back in the old day of WOTLK. (Link to the bluepost : https://www.wowhead.com/wotlk/news/wrath-classic-hotfixes-october-14-deep-wounds-and-fiery-weapon-329368 ) as proved by some old Wowhead comments such as this one : https://www.wowhead.com/item=33866/stormchops#comments:id=624481 . 

Therefore it has been changed on retail classic WOTLK but it shouldn't have been on Chromiecraft. Yet, when i'm testing this on the live server or on the PTR, none of the items mentionned above are able to proc Deep wounds when dealing a critical strike. 

This is due to spell crit and melee crit behind normalized and merged in WOTLK.

So it would be nice to revert this change and make deep wounds able to be procced from spell strike aswell, since Chromiecraft is made to have the most accurate scripting possible, not including blizzard classic retail changes.

Have a good day ! 

EDIT : typos.
EDIT 2 : Just noticed i put this topic in the wrong section. It should be moved into unresolved issues.

**Comment by BeefSteaker** [comment]
If you can find a source that it's blizzlike, put it in issues
https://github.com/chromiecraft/chromiecraft/issues/new/choose

**Reply by LiaohuChromie** [reply]
Done ! 

I'm new to the whole github thing and i wasn't able to delete this thread. So i just recreated it in the proper format in the issue section !
